src/main/java/core/basesyntax/strategy/filestrategy/impl/CsvReportBuilderImpl.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.strategy.filestrategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.FruitTransaction.Operation;
+import core.basesyntax.strategy.filestrategy.ReportBuilder;
+import core.basesyntax.strategy.operationstrategy.OperationCalculator;
+import core.basesyntax.strategy.operationstrategy.OperationCalculatorHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class CsvReportBuilderImpl implements ReportBuilder {
+    private static final String HEADER = "fruit,quantity";
+    private static final String DATA_SEPARATOR = ",";
+    private static final OperationCalculatorHandler operationHandler
+            = new OperationCalculatorHandler();
```

Comment:
```suggestion
    private final OperationStrategy operationStrategy;
```
initialize this field using constructor, implementation should be passed as a parameter to the constructor